stdlib/3/asyncio/futures.pyi

```diff
@@ -32,6 +32,7 @@ class Future(Awaitable[_T], Iterable[_T]):
     _blocking = False
     _log_traceback = False
     _tb_logger: Type[_TracebackLogger]
+    _loop: AbstractEventLoop
```

Comment:
From the C code it looks like this can be None. Should it be `Optional[AbstractEventLoop]`?

Also, there's no setter (assigning to `_loop` is not allowed), so this should be an `@property`.

Reply:
I think the constructor ensures that `_loop` cannot be `None`; however, the constructor does take in `loop` as `None.  If that's the case, we'll call `get_event_loop` to instantiate a loop.  I think `AbstractEventLoop` is the proper annotation because `Future.get_loop`uses the same one and it's implementation is returning `self._loop`

But, making this a `@property` is a great idea.  I'll do that!

Reply:
Looking at the C code again, it does look very hard to have it actually be None unless something went horribly wrong during initialization, so keeping the Optional off makes sense.